### PR TITLE
Finalise initial setup

### LINKED ISSUE #1: Finalise initial setup
In conjunction with https://github.com/nordic-dev-net/hydrophonitor/issues/22

- Configure depth-recorder service to start after multi-user.target is reached
- Implement conversion from ADC reading to pressure to depth

### PR DESCRIPTION
**Pull request by satukoskinen**
- Configure depth-recorder service to start after multi-user.target is reached
- First draft in implementing mapping from ADC output value to voltage to pressure to depth
- Values recorded to file are timestamp, voltage, pressure, and depth
- Log measured ADC value

Closes #1

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -86,12 +95,15 @@ fn main() {
             "ADC value: {}, converted voltage: {}, calculated pressure: {}, calculated depth: {}",
             value, voltage, pressure, depth
         );
-        let _ = writer.serialize(DepthData {
-            timestamp: Utc::now(),
-            voltage: voltage,
-            pressure: pressure,
-            depth: depth,
-        }).expect("Failed to serialize and write data to file");
+        let _ = writer
```
Comment (juliuskoskela) [inline src/main.rs]: Why do we have this `let`bindning here?

File: src/main.rs [hunk 2]
```diff
@@ -25,29 +25,38 @@ struct Cli {
...
+    // Map voltage in range 0-3.3V to pressure in range 0-100psi
+    let pressure_range_start: f32 = 0.0;
+    let pressure_range_end: f32 = 100.0;
+    let mapped_pressure_psi = (mapped_voltage - voltage_range_start)
+        * (pressure_range_end - pressure_range_start)
+        / (voltage_range_end - voltage_range_start);
+
+    // Convert pressure from psi to Pa, then calculate depth
+    let pressure_pascal = mapped_pressure_psi * 689.47573;
     let saltwater_density = 1023.6;
```
Comment (juliuskoskela) [inline src/main.rs:56]: Could this be a CLI argument to make the setup more general?

File: src/main.rs (hunk 1 above)
Comment (satukoskinen) [reply src/main.rs]: This is now cleaned up and removed

File: src/main.rs (hunk 2 above)
Comment (satukoskinen) [reply src/main.rs:56]: Yes, would make sense to have the option to pass these constants through CLI. Also voltage range and possibly pressure range. We could do that in a separate issue.

**Pull request merged**